app/services/scrapers/parties/high_priority.rb

```diff
@@ -0,0 +1,25 @@
+module Scrapers
+  module Parties
+    class HighPriority
+      def self.perform
+        new.perform
+      end
+
+      def perform
+        parties_oscn_ids = ::Party.without_html.limit(parties_high_count).pluck(:oscn_id)
+
+        bar = ProgressBar.new(parties_oscn_ids.count)
+        puts "#{parties_oscn_ids.count} are missing html"
+
+        parties_oscn_ids.each do |oscn_id|
+          PartyWorker.perform_async(oscn_id)
+          bar.increment!
+        end
+      end
+
+      def parties_high_count
+        ENV.fetch('PARTIES_HIGH_PRIORITY', 5_000).to_i
```

Comment:
This can be improved... might want to discuss a larger mechanism for limiting/optimizing request count

Reply:
When I have tech debt that I want to get to later, I usually just log it in an issue or JIRA card